5calls/app/src/main/java/org/a5calls/android/a5calls/controller/StatsActivity.java

```diff
@@ -97,15 +111,15 @@ private void initializeUI(DatabaseHelper db) {
         List<Long> contacts = db.getCallTimestampsForType(Outcome.Status.CONTACT);
         List<Long> voicemails = db.getCallTimestampsForType(Outcome.Status.VOICEMAIL);
         List<Long> unavailables = db.getCallTimestampsForType(Outcome.Status.UNAVAILABLE);
-        
+
         Spannable contactString = getTextForCount(contacts.size(),
-                R.string.impact_contact_one, R.string.impact_contact, R.color.contacted_color);
+                R.string.impact_contact_one, R.string.impact_contact, R.color.colorPrimaryDark);
```

Comment:
Optional: Do you think we should still construct this string, other than for the content description for the pie chart? It might be redundant.

Reply:
Also you don't need to use spannables if we aren't changing the colors. You can concat them like a normal string.